MekHQ/src/mekhq/campaign/personnel/turnoverAndRetention/Fatigue.java

```diff
@@ -143,7 +148,7 @@ public static int checkFieldKitchenUsage(List<Person> activePersonnel, boolean i
      * @return {@code true} if the available capacity is sufficient; {@code false} otherwise.
      */
     public static boolean areFieldKitchensWithinCapacity(int fieldKitchenCapacity, int fieldKitchenUsage) {
-        return fieldKitchenCapacity <= fieldKitchenUsage;
+        return fieldKitchenCapacity >= fieldKitchenUsage;
```

Comment:
```suggestion
        return fieldKitchenCapacity > fieldKitchenUsage;
```
I believe that it should only not be within capacity once larget than, not larger or equals to.

Reply:
No, larger than or equal to is correct. If you have 5 capacity you're still within capacity if you have 5 consumption.